src/redux/slices/meetingSlice.js

```diff
@@ -1,19 +1,296 @@
 import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
 import axios from 'axios';
+const API_URL = import.meta.env.VITE_API_URL || 'http://localhost:3000';
+
+export const createMeeting = createAsyncThunk(
+  'meeting/createMeeting',
+  async ({ token, newMeeting }, { rejectWithValue }) => {
+    try {
+      const { data } = axios.post(API_URL + '/api/meeting', newMeeting, {
+        headers: {
+          authorization: token,
+        },
+      });
+      return data;
+    } catch (error) {
+      console.log(error);
+      return rejectWithValue(error);
+    }
+  }
+);
+
+export const updateMeeting = createAsyncThunk(
+  'meeting/updateMeeting',
+  async ({ token, meetingId, meetingUpdates }, { rejectWithValue }) => {
+    try {
+      const { data } = axios.put(
+        API_URL + `/api/meeting/${meetingId}`,
+        meetingUpdates,
+        {
+          headers: {
+            authorization: token,
+          },
+        }
+      );
+      return data;
+    } catch (error) {
+      console.log(error);
+      return rejectWithValue(error);
+    }
+  }
+);
+
+export const getMeeting = createAsyncThunk(
+  'meeting/getOne',
+  async ({ token, meetingId }, { rejectWithValue }) => {
+    try {
+      const { data } = axios.get(API_URL + `/api/meeting/${meetingId}`, {
+        headers: {
+          authorization: token,
+        },
+      });
+      return data;
+    } catch (error) {
+      console.log(error);
+      return rejectWithValue(error);
+    }
+  }
+);
+
+// Route does not return the meeting that was deleted so the ex reducer won't be able to filter it out unless page is refreshed
```

Comment:
Is this comment still correct? Looks like you're filtering out by meetingId in the ER